File: level1/ex34.js
function solution(a, b) {
  // 요일 맵
  const dayMap = {
      0: 'SUN',
      1: 'MON',
      2: 'TUE',
      3: 'WED',
      4: 'THU',
      5: 'FRI',
      6: 'SAT',
  };

  // 요일 정보
  const day = new Date(`2016-${a}-${b}`).getDay();
  // 요일 정보 매핑
  return dayMap[day];
}

File: level1/ex34.py
import datetime

def solution(a, b):
    # 2016년 a월 b일
    date = datetime.datetime(2016, a, b)
    # 2016년 a월 b일의 요일 정보(영문)
    day = date.strftime('%A')
    # 요일 정보 3글자로 줄이고 대문자로 변환
    return day[:3].upper()